packages/blade/src/components/StepGroup/StepLine.web.tsx

```diff
@@ -24,7 +25,7 @@ const useDottedLineStyles = ({ isHorizontal }: { isHorizontal?: boolean } = {}):
   </svg>
   `;
 
-  const backgroundURL = `url(data:image/svg+xml;base64,${window.btoa(svgString)})`;
+  const backgroundURL = `url(data:image/svg+xml;base64,${encode(svgString)})`;
```

Comment:
need this library because we don't have any server build so we can't separate out `import Buffer from 'buffer'` which is needed to base64 conversion in node into separate build. Checks dont work because import itself needs to be differet.

This library handles that.

Reply:
we can but we'll also have to make changes in our build step. We handle this in universe but in blade repo we're only handling web and native